packages/@react-aria/select/src/useSelect.ts

```diff
@@ -85,7 +85,14 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
   let valueId = useId();
 
   return {
-    labelProps,
+    labelProps: {
+      ...labelProps,
+      onClick: () => {
+        if (!(<HTMLInputElement>ref.current).disabled) {
```

Comment:
I think we can just take isDisabled off the props?

Reply:
Hmm looks like babel is failing to parse this line. TBH I hadn't seen this TS syntax before either haha

Reply:
@snowystinger Will do 👍 

Me neither, I did it the 'normal' way but babel was failing to parse that too so thought id just give this a shot